.eslintrc.json

```diff
@@ -44,7 +44,6 @@
       }
     ],
     "eol-last": ["error", "always"],
-    "indent": ["error", 4],
```

Comment:
why are you removing this rule?

Reply:
I keep getting errors with it. It's doesn't go away when I use prettier and even run I run `lint`

Reply:
hmmm...okay